src/components/wishlistCard/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {View,Text,Image} from 'react-native'
+import Icon from 'react-native-vector-icons/FontAwesome'
+import styles from "./styles";
+import {Button} from 'components'
+import { NavigationContainer } from "@react-navigation/native";
+
+const WishlistCard = ({item}) =>{
```

Comment:
You haven't defined what will be the type of the item Prop.

Reply:
I will update after removing all the extra provers and other declarations.